modules/solid_mechanics/doc/content/modules/solid_mechanics/ShellElements.md

```diff
@@ -63,13 +63,13 @@ g^2=\dfrac{g_3 \times g_1}{g_1 \cdot (g_2 \times g_3)}
 \begin{equation}
 g^3=\dfrac{g_1 \times g_2}{g_1 \cdot (g_2 \times g_3)}
 \end{equation}
-These contravariant base vectors satisfy the orthogonality condition: $g^i \cdot g_j =\delta_{ij}$, where $\delta_{ij}$ is Kronecker delta. The $\hat{e}_i$ vectors are the local cartesian coordinate system computed using the covariant base vectors, i.e.,
+These contravariant base vectors satisfy the orthogonality condition: $g^i \cdot g_j =\delta_{ij}$, where $\delta_{ij}$ is Kronecker delta. The $\hat{e}_i$ vectors are the local cartesian coordinate system computed using the following rule:
```

Comment:
Well, the Y-axis (0, 1, 0 ) cannot really be the default value for `reference_first_local_direction` parameter. I think, we need an if condition anyway? The if condition is necessary because the first local vector is calculated differently when the user defines the reference direction vector. By default, the first local vector is determined by the **cross product** of the Y-axis with the element's normal vector. However, if the user provides a reference vector, the given vector is directly projected using a **dot product instead of a cross product**.  Please see my explanation in the .md file